modules/common-attributes.adoc

```diff
@@ -28,10 +28,12 @@ endif::[]
 :mtc-short: MTC
 :mtc-full: Migration Toolkit for Containers
 :mtc-first: Migration Toolkit for Containers ({mtc-short})
-:mtc-operator: MTC Operator
+:mtc-operator: {mtc-short} Operator
 :mtc-version: 1.3
-:mtc-version-z: 1.3.0
-:mtc-rhcam-folder: rhcam-1-3
-// image names to be confirmed
-:mtc-ocp3-download: registry.redhat.io/rhcam-1-3/openshift-migration-rhel7-operator:v1.3
-:mtc-must-gather: registry.redhat.io/rhcam-1-3/openshift-migration-must-gather-rhel8
+:mtc-version-z: v1.3.0
+:mtc-image-folder: rhmtc
+:mtc-registry: registry.redhat.io
```

Comment:
If the registry and namespace are not likely to change frequently, I question why they need to be attributes. I don't see how they're changing in different environments, so I don't think they're good candidates for replacement.

Reply:
The registry and namespace have changed in the past. Maybe they're stable enough now not to require attributes.